scripts/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt
+
+status=$?
+
+if [ "$status" = 0 ] ; then
+  echo "Lint completed successfully."
+  exit 0
+else
+  echo "********************************"
+  echo "Ktlint issue found."
+  echo "Please fix the above issues."
```

Comment:
Maybe output the suggested auto-fix line to make this easier for contributors. I personally always use ``-F`` and only fix the things ktlint can't.

Reply:
This will be helpful. I will add it.

Reply:
Done